shop/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+spring.datasource.url=jdbc:postgresql://127.0.0.1:5432/shop
+spring.datasource.username=postgres
+spring.datasource.password=postgres
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto=validate
```

Comment:
Read a little bit about what other values we can use for this property and which one is the best for development.
Reference: https://stackoverflow.com/questions/42135114/how-does-spring-jpa-hibernate-ddl-auto-property-exactly-work-in-spring